waspc/src/Wasp/Generator/JsImport.hs

```diff
@@ -1,26 +1,34 @@
 module Wasp.Generator.JsImport
-  ( getJsImportDetailsForExtFnImport,
+  ( mkJsImportGetterFromExtSrcDir,
+    RelDirToExternalCodeDir,
   )
 where
 
-import StrongPath (Dir, Path, Posix, Rel, (</>))
+import StrongPath (Dir, Dir', Path, Posix, Rel, (</>))
 import qualified StrongPath as SP
-import qualified Wasp.AppSpec.ExtImport as AS.ExtImport
+import qualified Wasp.AppSpec.ExtImport as EI
 import Wasp.Generator.ExternalCodeGenerator.Common (GeneratedExternalCodeDir)
+import Wasp.JsImport
+  ( JsImport,
+    JsImportName (JsImportField, JsImportModule),
+    getJsImport,
+  )
+
+type RelDirToExternalCodeDir = Path Posix (Rel GeneratedExternalCodeDir) Dir'
 
-getJsImportDetailsForExtFnImport ::
-  -- | Path to generated external code directory, relative to the directory in which file doing the importing is.
-  Path Posix (Rel a) (Dir GeneratedExternalCodeDir) ->
-  AS.ExtImport.ExtImport ->
-  -- | (importIdentifier, importStmt)
-  --   - importIdentifier -> Identifier via which you can access ext js function after you import it with importStmt.
-  --   - importStmt -> Javascript import statement via which you should do the import.
-  (String, String)
-getJsImportDetailsForExtFnImport relPosixPathToExtCodeDir extImport = (importIdentifier, importStmt)
+-- | Returns a getter function that can be used to get a JsImport from an ExtImport
+mkJsImportGetterFromExtSrcDir ::
+  Path Posix (Rel ()) (Dir GeneratedExternalCodeDir) ->
+  ( RelDirToExternalCodeDir ->
+    EI.ExtImport ->
+    JsImport
+  )
+mkJsImportGetterFromExtSrcDir rootDir relDirToExternalCodeDir extImport = getJsImport importPath importName
```

Comment:
I've named it `pathToExternalCodeDir` since the `relPathFromImportingJsFileToExternalCodeDir` is something we get when we combine `pathToExternalCodeDir` and the second argument `relDirToExternalCodeDir`